### PR TITLE
Complete full website to publish 

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this project, I created a full website for the Math magicians app, consisting of several pages and using the components I already created. In previous projects, I created the core functionality of the app: a calculator component. In this part I developed a full website with several pages. I created a website consisting of 3 pages: Home, Calculator, and Quote. For the purpose, I followed these steps: 

1. Add your own styles to improve the look and feel.
2. In the lesson about styling, we introduced several ways to add styles in a React application:
3. React basic styling (inline styles, importing CSS and pre-processed CSS files).
4. CSS modules.
5. Styled components.
6. In this project, you should use React basic styling.
7. For every page, you should create a route.
8. Make sure that navigation links for all 3 routes are displayed on each page.
9. Use React Router to make the routes work.

### CODE REVIEW

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi @tacettinkustu 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that lacks a pull request description. Please fix this and request for another review. Just briefly describe what you did in the milestone and you'll be done.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
+          <li key={path}>
+            <NavLink exact activeClassName="active" to={path}>
+              {name}
+            </NavLink>
+          </li>
+        ))}
+      </ul>
+    </nav>
+  </header>
+);
```
Comment (mikenath223) [inline src/App.js:41]: - [ ] OPTIONAL - Consider moving this header component to the component folder. It ensures that each component is kept in separate modules.

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,60 @@
...
+const App = () => (
+  <Router basename={process.env.PUBLIC_URL}>
+    <>
+      <Header />
+      <div className="container">
+        <Switch>
+          {routes.map(({ path, component }) => (
+            <Route exact key={path} path={path}>
+              {component}
+            </Route>
```
Comment (mikenath223) [inline src/App.js:52]: - Nice idea mapping routes this way. 👍

**Pull request merged**